development/tools/cmd/vmscollector/README.md

```diff
@@ -2,18 +2,18 @@
 
 ## Overview
 
-This command finds and removes orphaned virtual machines (VMs) created by the `kyma-integration` job in a Google Cloud Platform (GCP) project.
+This command finds and removes orphaned virtual machines (VMs) created by Prow jobs in a Google Cloud Platform (GCP) project.
 
-The `kyma-integration` job creates a VM instance to install and test Kyma.
+Prow jobs create a VM instance to install and test Kyma.
 Usually, the job also cleans up the instance.
 It can happen, however, that the job is terminated before its clean-up finishes.
 This causes a resource leak that generates unwanted costs.
 The garbage collector finds and removes such VM instances.
 
 There are three conditions used to find instances for removal:
-- The instance name pattern that is specific for the `kyma-integration` job
-- The value of a `job-name` label the instance is annotated with
-- The instance `creationTimestamp` value that is used to find instance existing at least for a preconfigured number of hours
+- The instance name is not caught by the exclude names regexp.
```

Comment:
```suggestion
- The instance name is not caught by the exclude names regex.
```

Reply:
According to Google, it's `regex`, not `regexp` 🤔 
Could you verify and change it in the rest of the document if needed?

Reply:
Done